x-pack/plugins/apm/server/index.ts

```diff
@@ -69,6 +69,80 @@ const configSchema = schema.object({
     schema.string({ defaultValue: '' }),
     schema.never()
   ),
+  featureFlags: schema.object({
```

Comment:
Did we decide not to merge the flags in the config from the list we were maintaining ?